docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -152,6 +156,38 @@ To manage and set the cookie options, use the [Tizen.WebView.CookieManager](http
     cookieManager.SetPersistentStorage(DirectoryInfo.Data, CookiePersistentStorage.SqlLite);
     ```
 
+<a name="eval"></a>
+## Executing JavaScript
+
+To execute the JavaScript code, use the `Tizen.WebView.Eval` and `Tizen.WebView.EvalAsync` methods.
+
+The following example shows the HTML file used:
+
+
+```
+<html>
+<body>
+    <div id="main" class="page">
+        <div class="contents">
+            <span id="content-text">Basic</span>
+        </div>
+    </div>
+</body>
+</html>
+```
+
+- If you need a return value, use the `Tizen.WebView.EvalAsync` method:
+
+    ```
+    string result = await webview.EvalAsync("document.getElementById('content-text').innerHTML");
+
```

Comment:
Question. In this code, Don't we need to use webView instead of webview?
(V in the capital letter.)

Also, would you please remove the one empty line 183, after the code line?

Reply:
Done